eureka/S5_lightcurve_fitting/likelihood.py

```diff
@@ -30,14 +30,19 @@ def ln_like(theta, lc, model, pmin, pmax, freenames):
 
     - December 29-30, 2021 Taylor Bell
         Moved code to separate file, added documentation.
+    - January 22, 2022 Megan Mansfield
+        Adding ability to do a single shared fit across all channels
     """
     # params[ifreepars] = freepars
     ilow = np.where(theta < pmin)
     ihi = np.where(theta > pmax)
     theta[ilow] = pmin[ilow]
     theta[ihi] = pmax[ihi]
     model.update(theta, freenames)
-    model_lc = model.eval()
+    if lc.share:
```

Comment:
Can we remove this if statement by passing `share=lc.share` and having `lc.longparamlist=None` by default?

Reply:
Fixed